tests/assets/dynamic-addon-cards.test.js

```diff
@@ -444,6 +458,29 @@ describe(__filename, () => {
         });
       });
 
+      it('disables the install button when add-on is not an extension', async () => {
```

Comment:
Nit: Should the test description include something about "on android"?

Reply:
Yes, thanks.